### ISSUE TITLE
JsPysch Record Video

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#1284]
# TL;DR
Record video of participant and save it to current EFP S3 store. 

# Narrative
EFP currently has the option of recording participants while they work through a study.  Additionally, it will also record a consent video.  Both of these videos are saved in a S3 store and a database entry is added to the Video table.  These two trials and a trial to set up their camera and microphone will need to be added to the CHS jsPych NPM package that was created from #1282.

# Acceptance Criteria
- [ ] Documentation
- [ ] Tests
- [ ] Record video/audio
- [ ] Save file to S3 Store via multipart upload
- [ ] Database entry for video file
- [ ] Custom trials for consent, start/stop video, video config

# Implementation Notes
We'd like to remove the different from trial video recording and session record.  It would be nice to have it all be the same thing.  The user should be able to start/stop the video at their whim.
